tests/chainerx_tests/unit_tests/testing_tests/test_dtypes.py

```diff
@@ -5,16 +5,34 @@
 
 
 def test_float_dtypes():
```

Comment:
The test name should be changed to something like that `test_dtypes_list`.

Reply:
Fixed.